kardinal-cli/cmd/root.go

```diff
@@ -853,7 +853,7 @@ func createDevFlow(tenantUuid api_types.Uuid, pairsMap map[string]string, templa
 	if resp.StatusCode() == 404 {
 		fmt.Printf("Could not create flow, missing %s: %s\n", resp.JSON404.ResourceType, resp.JSON404.Id)
 	} else if resp.StatusCode() == 500 {
-		fmt.Printf("Could not create flow, error %s: %v\n", resp.JSON500.Error, resp.JSON500.Msg)
+		fmt.Printf("Could not create flow, error %s: %s\n", resp.JSON500.Error, *resp.JSON500.Msg)
```

Comment:
I can add a sanity check around the pointer but it is safe looking at the server.